src/const_choice.rs

```diff
@@ -186,7 +186,7 @@ impl PartialEq for ConstChoice {
 #[derive(Debug, Clone)]
 pub struct ConstCtOption<T> {
     value: T,
-    is_some: ConstChoice,
+    pub(crate) is_some: ConstChoice,
```

Comment:
This isn't great but I'm not sure there's a better option without something like `const_mut_refs`